e2e/react/src/react.test.ts

```diff
@@ -156,28 +156,18 @@ forEachCli('nx', () => {
         JSON.stringify(
           {
             presets: ['@nrwl/react/babel'],
+            plugins: [
```

Comment:
Moved it up to the default plugins section since the jest sets `NODE_ENV=test`, which would override babel `envName`.